### ISSUE TITLE
RefreshDatabaseFast is seeding before migrating.

### ISSUE THREAD

**Issue by manu144x** [laracraft-tech/laravel-useful-additions#8]
I couldn't figure out why seeding was not working, but in the runMigrationsIfNecessary method of the Trait, you are first seeding (upon checking the configuration parameter, that's correct) but then you're doing a migrate fresh completely emptying the seeding :)

**Comment by Sairahcaz** [comment]
Please specify your issue more detailed, if you have further questions.
For instance give some example.


For now I'm gonna close this issue...

**Comment by stephenr85** [comment]
I believe this is in reference to RefreshDatabaseFast trait:

```php
/**
     * Check if migration fresh is necessary by checking
     * if the migrations files checksum has changed.
     */
    protected function runMigrationsIfNecessary(): void
    {
        if (! $this->identicalChecksum()) {
            if (config('useful-additions.refresh_db_fast.seed')) {
                $this->seed();
            }

            $this->artisan('migrate:fresh', $this->migrateFreshUsing());

            //create checksum after migration finishes
            $this->createChecksum();
        }
    }
```

I'm not using this feature, but also noticed this while reading the code for another issue.

**Comment by holmesadam** [comment]
I too have come across this issue - but I am using the feature. 

`laracraft-tech/laravel-useful-additions/src/Traits/RefreshDatabaseFast.php:42`

There is a check to see if the migrations need to run based on the checksum. With the config, you can have it seed too. However, the code seeds and then does a migrate fresh which effectively wipes out the seed. The seed should come after the migrate. See comment above for code block which I'm referring to.

**Comment by holmesadam** [comment]
I'm submitted a pull request to fix this #18
